overlord/snapstate/booted.go

```diff
@@ -94,7 +89,9 @@ func UpdateRevisions(ovld *overlord.Overlord) error {
 		for snapName, snapState := range installed {
 			if name == snapName {
 				if rev != snapState.Current {
-					ts, err := snapstate.RevertToRevision(st, name, rev, snapstate.Flags(0))
+					// FIXME: check that there is no task
```

Comment:
This is already done. We just need to make the error have a proper type and enough metadata so we can tell we're ignoring the right thing here.